students/dowoon/users/validators.py

```diff
@@ -0,0 +1,31 @@
+import re
+
+from django.core.exceptions import ValidationError
+
+from users.models import User
+
+
+def validate_email(email):
+    email_match_result = re.match("^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$", email)
+    if email_match_result == None:
```

Comment:
파이썬은 `True or False`를 판단할 수 있는 경우 굳이 불필요하게 None 객체와 비교하지 않고 아래와 같이 작성할 수 있습니다.
그리고 굳이 변수 지정까지 할 필요가 없어보이니 생략해줘도 될 것 같습니다. 모든 `validation`에 동일하게 적용해주세요
```suggestion
    if not re.match("^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$", email):
```

Reply:
확인 / 수정했습니다.